docs/contributor/05-10-e2e_tests.md

```diff
@@ -165,6 +166,36 @@ The test executes the following steps:
     make skr-binding-test
     ```
 
+## Provisioning Service Tests
+
+### Usage
+
+The test executes the following steps:
+
+1. Sends a call to Provisioning API to provision a Kyma runtime. The test waits until the environment is created.
+2. Creates a Kyma Binding.
+3. Fetches the `sap-btp-manager` secret using kubeconfig from created Kyma Binding.
+4. Fetches a created Kyma Biding.
+5. Deletes created Kyma Binding.
+6. Tries to fetch a `sap-btp-manager` secret using invalidated kubeconfig.
+7. Tries to fetch a deleted Kyma Binding.
+8. Sends a call to Provisioning API to deprovision a Kyma runtime. The test waits until the environment is deleted.
+
+### Test Execution
+
+1. Before you run the test, prepare the `.env` file based on this [`.env.template`](/testing/e2e/skr/provisioning-service-test/.env.template).
+2. To set up the environment variables in your system, run:
+
+    ```bash
+    export $(xargs < .env)
+    ```
+
+3. Run the test scenario:
+
+    ```bash
+    make provisioning-service
+    ```
+
 ## CI Pipelines
 
 The tests are run once per day at 01:05 by the given ProwJobs:
```

Comment:
It is not part of your changes, but is this still valid?

Reply:
This is not valid, tests are run once per day starting at 9 at hourly intervals. I will change it.